src/medCore/gui/toolboxes/medToolBox.cpp

```diff
@@ -64,6 +65,8 @@ medToolBox::medToolBox(QWidget *parent) : QWidget(parent), d(new medToolBoxPriva
     connect(d->body, SIGNAL(maximized()), this, SLOT(behaveWithBodyVisibility()));
 
     this->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Minimum);
+
+    connect(this, SIGNAL(displayError(int)), this, SLOT(handleDisplayError(int)));
```

Comment:
OK, but in this file you are connecting a signal of the class with a slot of the same class, which .. you know :)
That's why I said it would be better to do in med..Tbx:
`connect(job, SIGNAL(displayError(int)), this, SLOT(handleDisplayError(int)));`
It would call the slot of medToolBox (no need to have the `displayError` signal in medToolBox)